File: sdks/java/src/main/java/org/contractgraphqa/interop/ConformanceReportException.java
package org.contractgraphqa.interop;

/** Raised when a conformance report is ambiguous, incomplete, or drifts from v0.1. */
public final class ConformanceReportException extends Exception {
    public ConformanceReportException(String message) {
        super(message);
    }

    public ConformanceReportException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: sdks/dotnet/src/ContractGraphQA.Interop/ConformanceReportException.cs
namespace ContractGraphQA.Interop;

/// <summary>Raised when a conformance report is ambiguous, incomplete, or drifts from v0.1.</summary>
public sealed class ConformanceReportException : Exception
{
    public ConformanceReportException(string message) : base(message) { }

    public ConformanceReportException(string message, Exception innerException)
        : base(message, innerException) { }
}
